appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/ModuleDescriptor.java

```diff
@@ -51,18 +54,27 @@
      * Returns the context root of the application if any
      * @return 
      */
-    public String getContextRoot();
+    String getContextRoot();
```

Comment:
Why has the visibility changed as this is a breaking change? These classes are exposed via the public Payara Micro api and now these methods aren't visible.

Reply:
ModuleDescriptor is an interface. Methods in an interface are public by default, not package or anything else. The public access modifier is redundant to use for interface methods. I.e. removing or adding them makes no difference for the visibility.